pinot-spi/src/main/java/org/apache/pinot/spi/env/CommonsConfigurationUtils.java

```diff
@@ -32,27 +32,54 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 import java.util.stream.StreamSupport;
-import org.apache.commons.configuration.ConfigurationException;
-import org.apache.commons.configuration.PropertiesConfiguration;
 import org.apache.commons.configuration2.Configuration;
+import org.apache.commons.configuration2.PropertiesConfiguration;
+import org.apache.commons.configuration2.convert.DefaultListDelimiterHandler;
+import org.apache.commons.configuration2.ex.ConfigurationException;
+import org.apache.commons.configuration2.io.FileHandler;
 import org.apache.commons.lang3.StringUtils;
 
 
 /**
  * Provide utility functions to manipulate Apache Commons {@link Configuration} instances.
  */
 public class CommonsConfigurationUtils {
+  private static final Character DEFAULT_LIST_DELIMITER = ',';
   private CommonsConfigurationUtils() {
   }
 
+  public static void setDefaultListDelimiterHandler(PropertiesConfiguration configuration) {
+    configuration.setListDelimiterHandler(new DefaultListDelimiterHandler(DEFAULT_LIST_DELIMITER));
+  }
+
+  public static void loadPropertiesConfiguration(PropertiesConfiguration configuration, String path)
```

Comment:
Same for other loading methods, we can directly return `PropertiesConfiguration`
```suggestion
  public static PropertiesConfiguration loadFromFile(String filePath)
```